app/views/admins/settings/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for [:admin, @settings], url: admin_settings_path do |f| %>
+  <%= f.collection_select :current_semester_id, Semester.order(:year), :id, :season_with_year, include_blank: true %>
+  <%= f.input :npo_app_open, label: 'NPO Application Status', collection: [ ['Open', 'Open' ], ['Closed', 'Closed'] ], as: :radio_buttons, label_method: :last, value_method: :first, checked: @settings.npo_app_open, item_wrapper_class: 'inline'%>
+  <%= f.input :student_app_open, label: 'Student Application Status', collection: [ ['Open', 'Open' ], ['Closed', 'Closed'] ], as: :radio_buttons, label_method: :last, value_method: :first, checked: @settings.student_app_open, item_wrapper_class: 'inline'%>
```

Comment:
Interesting. Does having that `collection` work? I thought that each item would have to look something like `["Open", true]` (eg. the future db value is in the array)